lib/common/client/concerns/mhv_locked_session_client.rb

```diff
@@ -19,8 +19,8 @@ module MhvLockedSessionClient
         extend ActiveSupport::Concern
         include SentryLogging
 
-        LOCK_RETRY_DELAY = 1 # Number of seconds to wait between attempts to acquire a session lock
-        RETRY_ATTEMPTS = 10 # How many times to attempt to acquire a session lock
+        LOCK_RETRY_DELAY = 0.3 # Number of seconds to wait between attempts to acquire a session lock
+        RETRY_ATTEMPTS = 30 # How many times to attempt await of acquiring a session lock by a preceding request
```

Comment:
The 0.3ms is okay, but I think we should make the total try-time > lock ttl (which is 10 seconds). Maybe make it retry 40 times, which is 12 seconds, so another thread has a chance to reacquire a stale lock.